### PR TITLE
Return of the dummy app

### PR DESCRIPTION
**Pull request by no-mercy**

### CODE REVIEW

File: tests/dummy/app/initializers/insights.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import Ember    from 'ember';
+import Insights from 'ember-insights';
+
+export default {
+
+  name: 'Google Analytics insights',
```
Comment (avdeevs) [inline tests/dummy/app/initializers/insights.js]: Cool name ;)

File: tests/dummy/app/initializers/insights.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    }).track({
+      insights: {
+        // ALL_TRANSITIONS: true
+        ALL_TRANSITIONS: {
+              except: ['index', 'main.record', 'outer', 'outer.inner', 'outer.inner.nested']
+            },
+        TRANSITIONS: ['index', 'outer.inner.nested'],
+        ACTIONS: ['testAction1'],
+        MAP: {
+              outer: {
```
Comment (avdeevs) [inline tests/dummy/app/initializers/insights.js]: Something happened to indentations

File: tests/dummy/app/initializers/insights.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+    // configure insights for certain environment
+    Insights.configure('development', {
+      debug: true,
+      trackerFactory: Insights.ConsoleTracker.factory
+    }).track({
+      insights: {
+        // ALL_TRANSITIONS: true
+        ALL_TRANSITIONS: {
+              except: ['index', 'main.record', 'outer', 'outer.inner', 'outer.inner.nested']
```
Comment (avdeevs) [inline tests/dummy/app/initializers/insights.js]: indentations

**Comment by Zatvobor** [comment]
merged to the rc #70

**Pull request merged**